man/mkdocs/overrides/partials/copyright.html

```diff
@@ -0,0 +1,17 @@
+<div class="md-copyright">
+    {% if config.copyright %}
+    <div class="md-copyright__highlight">
+        {{ config.copyright }}
+    </div>
+    {% endif %}
+    Partially supported by
+    <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=2303651" target="_blank" rel="noopener">
+        U.S. NSF Award (2303651)
```

Comment:
What everything thinks about "Modernization in part supported by US NSF"?

```suggestion
    Modernization in part supported by
    <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=2303651" target="_blank" rel="noopener">
        US NSF
```

Reply:
I just committed:

"Migration to Material for MkDocs supported by US NSF"